minimal-distributed.yaml

```diff
@@ -39,22 +46,38 @@ services:
       - "5062:5062"
   sprout:
     build: sprout
+    environment:
+      - ADDITIONAL_SHARED_CONFIG
     ports:
       - 22
```

Comment:
@johadalin Because this is just about exposing ports externally so that they can be contacted from outside of the deployment?   All SIP traffic goes via Bono rather than direct to Sprout right?